src/Grid/grid.jl

```diff
@@ -355,7 +422,30 @@ This function takes the local vertex representation (a `VertexIndex`) and looks
 toglobal(grid::AbstractGrid,vertexidx::VertexIndex) = vertices(getcells(grid,vertexidx[1]))[vertexidx[2]]
 toglobal(grid::AbstractGrid,vertexidx::Vector{VertexIndex}) = unique(toglobal.((grid,),vertexidx))
 
-@inline getdim(::AbstractGrid{dim}) where {dim} = dim
+getsdim(::AbstractGrid{sdim}) where sdim = sdim
+@inline getdim(g::AbstractGrid) = getsdim(g) # TODO: Deprecate
+
+"""
+    get_reference_dimensionality(grid::AbstractGrid) -> Union{Int, Symbol}
+
+Get information about the reference dimensions of the cells in the grid. 
+If all cells have the same reference dimension, `rdim::Int` is returned. 
+Otherwise, the `Symbol` `:mixed` is returned indicating a mixed-rdimensionality grid.
+"""
+get_reference_dimensionality(g::AbstractGrid) = _get_reference_dimensionality(getcells(g))
```

Comment:
`getrdim() `?

Reply:
Would be ok. Had that first, then `getrdims()::Tuple` since it could be different. But never needed the information about which rdims are in the grid if more than one, so ended with returning `:mixed` for that case. I didn't want to error on :mixed, and calling `rdim` getting a Symbol with just the information was confusing to me (but potentially only to me).

Reply:
Renamed to `get_reference_dimension`